README.md

```diff
@@ -12,8 +12,51 @@ Please, post an issue on GitHub and **provide a plunker** with your question.
 ```javascript
 npm install ng2-eonasdan-datetimepicker --save
 ```
-
-2) Add the A2Edatetimepicker module import in your module.:
+2) Include all required scripts if it doesn't included yet in your project:
+ - jquery.js, 
+ - bootstrap (.js & .css),
+ - eonasdan bootsrap datetimepicker (.js & .css),
+ 
+ - moment.js.
+
+   (nb: you don't need to do another npm install, the previous command already did it)
```

Comment:
This note is obsolete.

Reply:
Want me to remove it or you'll do when merging?

Reply:
i will do it, thanks for all the support.